redfish-core/schema/oem/openbmc/json-schema/OemLogEntryAttachment.json

```diff
@@ -48,12 +48,17 @@
                     "description": "PEL in Json format.",
                     "readonly": true,
                     "type": ["string", "null"]
+                },
+                "AdditionalDataFullAuditLogURI": {
+                    "description": "Additional data of AuditLog.",
```

Comment:
Suggest making description match between xml and json file.

Reply:
Corrected